types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,197 @@ declare namespace googleMapReact {
+            APPROXIMATE: string;
+            GEOMETRIC_CENTER: string;
+            RANGE_INTERPOLATED: string;
+            ROOFTOP: string;
+        };
+        GeocoderStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            ERROR: string;
+        };
+        KmlLayerStatus: {
+            DOCUMENT_NOT_FOUND: string;
+            DOCUMENT_TOO_LARGE: string;
+            FETCH_ERROR: string;
+            INVALID_DOCUMENT: string;
+            INVALID_REQUEST: string;
+            LIMITS_EXCEEDED: string;
+            OK: string;
+            TIMED_OUT: string;
+            UNKNOWN: string;
+        };
+        MapTypeControlStyle: {
+            DEFAULT: number;
+            DROPDOWN_MENU: number;
+            HORIZONTAL_BAR: number;
+            INSET: number;
+            INSET_LARGE: number;
+        };
+        MapTypeId: {
+            HYBRID: string;
+            ROADMAP: string;
+            SATELLITE: string;
+            TERRAIN: string;
+        };
+        MaxZoomStatus: {
+            ERROR: string;
+            OK: string;
+        };
+        NavigationControlStyle: {
+            ANDROID: number;
+            DEFAULT: number;
+            Ot: number;
+            SMALL: number;
+            Vz: number;
+            ZOOM_PAN: number;
+        };
+        RenderingType: {
+            RASTER: string;
+            UNINITIALIZED: string;
+            VECTOR: string;
+        };
+        ScaleControlStyle: {
+            DEFAULT: number;
+        };
+        StreetViewPreference: {
+            BEST: string;
+            NEAREST: string;
+        };
+        StreetViewStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            ZERO_RESULTS: string;
+        };
+        StreetViewSource: {
+            DEFAULT: string;
+            OUTDOOR: string;
+        };
+        StrokePosition: {
+            CENTER: number;
+            INSIDE: number;
+            OUTSIDE: number;
+        };
+        SymbolPath: {
+            CIRCLE: number;
+            BACKWARD_CLOSED_ARROW: number;
+            BACKWARD_OPEN_ARROW: number;
+            FORWARD_CLOSED_ARROW: number;
+            FORWARD_OPEN_ARROW: number;
+        };
+        TrafficModel: {
+            BEST_GUESS: string;
+            OPTIMISTIC: string;
+            PESSIMISTIC: string;
+        };
+        TransitMode: {
+            BUS: string;
+            RAIL: string;
+            SUBWAY: string;
+            TRAIN: string;
+            TRAM: string;
+        };
+        TransitRoutePreference: {
+            FEWER_TRANSFERS: string;
+            LESS_WALKING: string;
+        };
+        TravelMode: {
+            BICYCLING: string;
+            DRIVING: string;
+            WALKING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
```

Comment:
```suggestion
            TRANSIT: string;
            TWO_WHEELER: string;
            WALKING: string;
```

Think that you forgot those properties when sorting alphabetically 😉

Reply:
🤦🏼